assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -44,22 +46,26 @@ import {
 import MediaElement from './mediaElement';
 
 const Container = styled.div`
+  grid-area: infinitescroll;
   display: grid;
   grid-gap: 10px;
   grid-template-columns: 1fr 1fr;
+  overflow: auto;
+  padding: 15px 1em 0 1em;
```

Comment:
Let's use `px` for all paddings. Especially for horizontal padding `em` are a bit confusing.

Reply:
I used matching values from the element above. It all will be reworked with a new design waiting in Figma.